app/assets/stylesheets/main.scss

```diff
@@ -0,0 +1,74 @@
+* {
+  --dark-sienna: rgba(34, 12, 16, 1);
+  --feldgrau: rgba(80, 108, 100, 1);
+  --middle-blue-green: rgba(119, 203, 185, 1);
+  --dark-sky-blue: rgba(117, 184, 200, 1);
+  --light-gray: rgba(205, 211, 213, 1);
+  box-sizing: border-box;
+}
```

Comment:
`scss` offers a way to assign variables with the `$` syntax:
```
$dark-sienna: rgba(34, 12, 16, 1);
```
as opposed to `var`

Reply:
We can submit a new one tomorrow correcting this mistake before we get too deep into other styles @dovakeen118